### ISSUE TITLE
cards: improve a11y alt text in generated img tag

### ISSUE THREAD

**Issue by jcharousset** [McShelby/hugo-theme-relearn#1203]
Hello,

Thank you for all the work you’ve put into this project — it’s impressive and very helpful.

The new "cards" shortcode is nice, and the "children" shortcode with "type=cards" even more. However the generated <img> tag doesn't have an alt text, which is flagged as an error by many HTML proofers.

A quick & easy solution could be to always add an empty alt text in the generated <img> tag. This sort of makes senses to me, as images in cards are frequently associated with a title and/or description, so they could be considered decorative. Maybe in the future this could be extended with a mechanism allowing to specify an alt text.
